src/main.ts

```diff
@@ -85,7 +85,8 @@ async function start () {
   }
 
   // start Vue application
-  console.info('Starting app...') // eslint-disable-line no-console
+  const aboutApp = import.meta.env.ABOUT_APP
+  console.info(`Starting ${aboutApp}...`) // eslint-disable-line no-console
```

Comment:
Thoughts?

![image](https://github.com/bcgov/business-create-ui/assets/10002889/fee1e26c-858c-4034-a3ff-62d4396ac6e1)

Reply:
I like it 😃